corehq/apps/reports/filters/users.py

```diff
@@ -410,3 +440,75 @@ def get_user_toggle(request):
     elif group or individual:
         show_filter = False
     return toggle, show_filter
+
+
+class ExpandedMobileWorkerFilterAllUsers(ExpandedMobileWorkerFilter):
```

Comment:
Since this is only used in the submit history report, I'd prefer to call it something like `SubmitHistoryFilter`.  The "all users" thing is only one aspect among many - the filter also includes groups, locations, and some meta options.  It feels weird to call it out in the name of the class - it's context that I'd put in the commit message and PR, and maybe a comment or docstring explaining why it's a separate class from the EMWF.

Reply:
Makes sense, I made the change to a bunch of classes like that here: https://github.com/dimagi/commcare-hq/pull/21564/commits/68e5eafd0dcf17a636e27ad047750bd0e4592440